routes/api/v1/forecast.js

```diff
@@ -3,22 +3,34 @@ var express = require('express');
 var router = express.Router();
 const fetch = require('node-fetch')
 
```

Comment:
I like how you have broken out the functionality here.  Looks like you are all set to organize these pieces of functionality into modules later on.

Reply:
Thanks Smitha!